### PR TITLE
Lgx stealthburner cable cover

### PR DESCRIPTION
**Pull request by sammynorway**
Connector cover that fits stealthburner with LGX extruder. 

I was searching for connector cover that could fit my stealthburner LGX toolhead.
I could not find a cover that would fit my toolhead, so I redesigned craxoor Afterburner PCB Cover. 
First time using git and sharing. I think I have done things right. Looking forward to get feedback 
Skjermbilde 2022-08-18 kl  12 42 01

### CODE REVIEW

File: printer_mods/sammynorway/LGX-stealthburner-connector-cover/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# LGX stealthburner connector cover
+
+I was searching for connector cover that could fit my stealthburner LGX toolhead.
+I could not find a cover that would fit my toolhead, so I redesigned [craxoor](https://github.com/craxoor/VoronMods/tree/master/PCB%20Cover) Afterburner PCB Cover. 
+
+![alt text](./Images/Preview-front.png)
```
Comment (MapleLeafMakers) [inline printer_mods/sammynorway/LGX-stealthburner-connector-cover/README.md:6]: `alt text` is a placeholder.  this should be replaced a textual description of the image.

File: printer_mods/sammynorway/LGX-stealthburner-connector-cover/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# LGX stealthburner connector cover
+
+I was searching for connector cover that could fit my stealthburner LGX toolhead.
+I could not find a cover that would fit my toolhead, so I redesigned [craxoor](https://github.com/craxoor/VoronMods/tree/master/PCB%20Cover) Afterburner PCB Cover.
```
Comment (MapleLeafMakers) [inline printer_mods/sammynorway/LGX-stealthburner-connector-cover/README.md:4]: Just a suggestion, maybe consider rewriting this as: 

```This is a remix of [craxoor's Afterburner PCB Cover](https://github.com/craxoor/VoronMods/tree/master/PCB%20Cover) designed to work with the Stealthburner and Bondtech's LGX Extruder.```

**Review by MapleLeafMakers — COMMENTED** [review]
Can you please rename the stl file to remove spaces.
if this part is intended to be printed in the accent color, please prefix the filename with `[a]_`  I would suggest `[a]_lgx_stealthburner_pcb_cover.stl`

File: printer_mods/sammynorway/LGX-stealthburner-connector-cover/README.md (hunk 2 above)
Comment (sammynorway) [reply printer_mods/sammynorway/LGX-stealthburner-connector-cover/README.md]: Thank you. This sounds a lot better

**Comment by MapleLeafMakers** [comment]
Sorry about the confusion, but since there are 2 parts in one STL File here, we have a convention to add `_set` at the end of the filename.  So `[a]_lgx_stealthburner_pcb_cover_set.stl`  
Otherwise this looks good to me!

**Review by cbc02009 — APPROVED** [review]

**Pull request merged**